molecule/users/molecule.yml

```diff
@@ -0,0 +1,49 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint:
+  name: yamllint
+platforms:
+  - name: jumpbox1
+    image: ubuntu:trusty
+    groups:
+      - jumpbox
+  - name: jumpbox2
+    image: ubuntu:bionic
+    groups:
+      - jumpbox
+provisioner:
+  name: ansible
+  lint:
+    name: ansible-lint
+  inventory:
+    group_vars:
+      jumpbox:
+        jumpbox_operators:
+          - username: inactiveuser
+            email: inactiveuser@example.com
+            public_key: ssh-rsa inactiveuser-public-key-string comment
+          - username: testuser
+            email: testuser@example.com
+            public_key: ssh-rsa testuser-public-key-string comment
+            active: true
```

Comment:
All the active/inactive functionality is still there, so there's no reason to remove, right? There might be a use case in the future where we want to remove someone temporarily...?

Reply:
But why not just delete them from the list in that case? At this point I'm not sure what the difference is between `active: false` and being removed from the list. It seems like a redundant feature, extra code to read, extra to test, extra docs, extra ways to typo and make a mistake.

Reply:
That's fair, there is no difference: both removing and setting active: false provide the same functionality, and having extra code to maintain in the future has been a pain point of ours. If we want to remove that functionality, it will need to be done in conjunction with a PR on datagov-deploy clearing out the current inactive user list. Could we setup a new ticket to remove this as tech debt, since [this feature](https://github.com/GSA/datagov-deploy/issues/1837) as described is done?

Reply:
The more I dig into #1203, the more I think it **would** be much easier to get rid of `active: false` functionality. I think I will do the coordinated update as a hotfix before deployment...